app/views/RoomView/index.js

```diff
@@ -110,6 +110,7 @@ class RoomView extends React.Component {
 		this.rid = props.route.params?.rid;
 		this.t = props.route.params?.t;
 		this.tmid = props.route.params?.tmid;
+		this.isThread = props.route.params?.t === 'thread' || false;
```

Comment:
No need to add `isThread`.
If `tmid` (aka thread message id) has content, it's a thread.